scoap3/users/api/views.py

```diff
@@ -14,9 +16,11 @@ class UserViewSet(RetrieveModelMixin, ListModelMixin, UpdateModelMixin, GenericV
     serializer_class = UserSerializer
     queryset = User.objects.all()
     lookup_field = "username"
+    permission_classes = [AllowAnyGetAuthenticatedOthers]
 
     def get_queryset(self, *args, **kwargs):
-        assert isinstance(self.request.user.id, int)
+        if self.request.user.id:
```

Comment:
I added this line here since the user is not authenticated. 
Is it what we want?